hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/AbstractBackendStoreProvider.java

```diff
@@ -131,10 +133,15 @@ public void clear() throws BackendException {
     }
 
     @Override
-    public void truncate() {
+    public void truncate(HugeGraph graph) {
         this.checkOpened();
+        HugeConfig config = (HugeConfig) graph.configuration();
+        String systemStoreName = config.get(CoreOptions.STORE_SYSTEM);
```

Comment:
can split `stores` into multi vars, like schemaStore + systemStore + graphStore

Reply:
tiny comment, address optionally

Reply:
I won't deal with it this time.